src/app/components/ContactModalRow.js

```diff
@@ -44,15 +44,22 @@ const ContactModalRow = ({ property, onChange, onRemove, onMoveUp, onMoveDown, f
 
     return (
         <Row>
+            {isOrderable && (
```

Comment:
I personnaly perfer:
```
{condition ? <Comp /> : null}
```

Reply:
I usually go with && mostly because it requires one to write conditions in more explicit manner and therefore easier to process (`arr.length` vs `arr.length > 0`). But I'm fine with both as well, so probably I will follow existing style where possible in the future.